docusaurus/docs/shared/create-plugin.md

```diff
@@ -17,13 +17,13 @@ The Grafana [create-plugin tool](https://www.npmjs.com/package/@grafana/create-p
 1. Install the dependencies:
 
    ```
-   yarn install
+   npm install
    ```
 
 1. Build the plugin:
 
    ```
-   yarn dev
+   npm dev
    ```
 
 1. Restart the Grafana server for Grafana to discover your plugin.
```

Comment:
The following line doesn't make sense for a generic plugin. We can _probably_ switch to admin > plugins and again use {props.pluginType} or just state "that your plugin is there"